### ISSUE TITLE
Cannot convert value of type 'RoomObjectAnchor' to expected argument type 'AnchoringComponent.Target'

### ISSUE THREAD

**Issue by danlee-lils** [jmousseau/RoomObjectReplicator#3]
hello... i have been getting this error any idea what is happening? 

public convenience init(_ anchor: RoomObjectAnchor) {
        self.init(dimensions: anchor.dimensions, category: anchor.category)
        components.set([AnchoringComponent(anchor)]) ==> error getting here
} 

This works on my other M1 mac... but not on older mac...

**Comment by jmousseau** [comment]
I just compile successfully on a 2019 MacBook Pro. Can you share more details about the target you are building for (i.e. "Any iOS Device"). Looking at the RealityKit initialize definition, I don't see anything wrong as we are passing an `ARAnchor` subclass.

```swift
@available(macOS 10.15, iOS 13.0, macCatalyst 14.0, *)
extension AnchoringComponent {

    /// Creates an anchoring component with the given AR anchor.
    ///
    /// - Parameters:
    ///   - anchor: An existing AR anchor to use.
    public init(_ anchor: ARAnchor)
}

```

**Comment by danlee-lils** [comment]
Thank you for the reply. It seems that the latest XCode has resolved the issue. I apologize for false alarm!
